src/test/java/teammates/test/cases/logic/FeedbackResponseCommentsLogicTest.java

```diff
@@ -278,7 +275,7 @@ public void testDeleteFeedbackResponseComment() throws Exception {
         frComment.feedbackResponseId = "invalid responseId";
         //without proper frCommentId and its feedbackResponseId,
         //it cannot be deleted
-        frcLogic.deleteFeedbackResponseComment(frComment);
+        frcLogic.deleteFeedbackResponseCommentById(1234567L);
 
         FeedbackResponseCommentAttributes actualFrComment =
```

Comment:
The purpose of the following verification is?

Reply:
Yes, This is not needed here. I think we can safely remove test case `silent fail nothing to delete`. Your thoughts in this?

Reply:
No we still need that test case but you don't need to add any verification. You can think of the verification as 'It doesn't throw exception'.